### PR TITLE
Add firestore

### PR DESCRIPTION
**Pull request by MacMalainey**
## Checklist

- [ ] Automated tests
- [ ] Screenshots included (if necessary)
- [ ] Code is well commented

Closes #ISSUE_ID

### CODE REVIEW

**Review by Docker-J — DISMISSED** [review]

File: src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/DatabaseService.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.cmput301f21t09.budgetprojectname.services.database;
+
+import com.google.firebase.database.FirebaseDatabase;
+
+public class DatabaseService {
+
+    private static final FirebaseDatabase db = FirebaseDatabase.getInstance();
```
Comment (j-vent) [inline src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/DatabaseService.java:7]: I think we were switching over to Firestore so should be FireBaseFireStore db = ...

File: src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/LocationDescriptor.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * Template for a description of a DatabaseLocation
+ */
+interface LocationDescriptor {
+
+    /**
+     * Return a DatabaseReference with the added description
+     * @param reference the reference to add description to
+     * @return reference with added description
+     */
+    DatabaseReference describe(DatabaseReference reference);
```
Comment (j-vent) [inline src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/LocationDescriptor.java:15]: Same here, should be CollectionReference

**Review by j-vent — COMMENTED** [review]
I'm not sure if this was an older PR so all my comments were to do with switching over from Database to Firestore

File: src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/DatabaseService.java (hunk 1 above)
Comment (MacMalainey) [reply src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/DatabaseService.java:7]: Whoops, I thought I switched branches

Comment (MacMalainey) [reply src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/DatabaseService.java]: Fixed

File: src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/LocationDescriptor.java (hunk 2 above)
Comment (MacMalainey) [reply src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/LocationDescriptor.java]: Removed as that was not supposed to be committed

**Review by Docker-J — APPROVED** [review]

**Review by j-vent — APPROVED** [review]

**Pull request merged**